### PR TITLE
trino init at 422

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

/cc @johnfosborneiii

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: trino.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      tar -C /home/build/trino-all/ --strip-components 1 -xf core/trino-server/target/trino-server-${{package.version}}.tar.gz
+
+      mvn package -q -DskipTests -T $(nproc) -pl ':trino-cli'
+      chmod +x client/trino-cli/target/trino-cli-${{package.version}}-executable.jar
+
+      # rsync -av --force /home/build/trino-all/plugin/jmx /home/build/plugins/default/
+      # rsync -av --force /home/build/trino-all/plugin/memory /home/build/plugins/default/
+      # rsync -av --force /home/build/trino-all/plugin/tpcds /home/build/plugins/default/
+      # rsync -av --force /home/build/trino-all/plugin/tpch /home/build/plugins/default/
+      # rsync -av --exclude memory --exclude tpcds --exclude tpch --exclude jmx /home/build/trino-all/plugin/* /home/build/plugins/optional/
```
Comment (joshrwolf) [inline trino.yaml:70]:
```suggestion
```

can these be removed?

File: trino.yaml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      rsync -av /home/build/trino-all/plugin/* /home/build/plugins/optional/
```
Comment (joshrwolf) [inline trino.yaml:71]: just curious why `rsync` instead of `cp`? do these come from some upstream build?

File: trino.yaml [hunk 3]
```diff
@@ -0,0 +1,165 @@
+package:
+  name: trino
+  version: 422
+  epoch: 0
+  description:
```
Comment (imjasonh) [inline trino.yaml:5]: Needs a description

File: trino.yaml [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  target-architecture:
+    - all
```
Comment (imjasonh) [inline trino.yaml:7]:
```suggestion
```

File: trino.yaml (hunk 1 above)
Comment (developer-guy) [reply trino.yaml]: for sure!

File: trino.yaml (hunk 2 above)
Comment (developer-guy) [reply trino.yaml:64]: to be honest, don't know much about that package, maybe @johnfosborneiii can answer that question, also I didn't find any `rsync` command anywhere on that project.

Comment (johnfosborneiii) [reply trino.yaml:64]: I switched from mv/cp to rsync due to some "rename" errors that I was getting, I ended up resolving the root cause and it was unrelated to mv/cp so you could switch it back if that would be more consistent or preferred.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**